packages/atomic/src/components/search/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -37,6 +37,7 @@ import {loadDayjsLocale} from '../../../utils/dayjs-locales';
 import {loadGlobalScripts} from '../../../global/global';
 import {BaseAtomicInterface} from '../../common/interface/interface-common';
 import {CommonBindings} from '../../common/interface/bindings';
+import {initi18n} from '../../common/interface/i18n';
```

Comment:
For this file I implemented the same changes in [my PR](https://github.com/coveo/ui-kit/pull/2145) with a bit more details, I'd appreciate if this file wasn't merged 🙏